### PR TITLE
Account logic - Closes #429

### LINKED ISSUE #429: Implement account logic in React
We need to have the logic of fetching and maintaining account information implemented. This includes all actions, reducers and every required utilities along with their unit tests.

### PR DESCRIPTION
**Pull request by reyraa**
- Adds actions to set and reset account details along with required utilities.
 - Add reducers to manage the state for account info.

### CODE REVIEW

File: src/components/counter.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { createStore } from 'redux';
 import { Provider, connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
+document.addEventListener('syncTick', e => console.log(e));
+
```
Comment (alepop) [inline src/components/counter.js]: Redux logger will show you all actions that will be triggered. Is this listener needed?

Comment (reyraa) [inline src/components/counter.js]: no, that was just for dev purpose which I forgot to delete

**Comment by slaweet** [comment]
Overall great work.
So far I noticed two issues:
- Jenkins passes even though there are 2 eslint errors in Jenkins
- `metronome.js` has low test coverage.

File: src/utils/polyfills.test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { expect } from 'chai';
+import { deepEquals } from './polyfills';
+
+describe('Polyfills', () => {
```
Comment (slaweet) [inline src/utils/polyfills.test.js]: Here I'm missing one level of describe:
`describe('deepEquals', () => {`

File: src/main.js [hunk 3]
```diff
@@ -2,9 +2,23 @@
 import React from 'react';
 import ReactDom from 'react-dom';
 import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
-import ReduxConter from './components/counter';
+import ReduxCounter from './components/counter';
+import Metronome from './utils/metronome';
 
-const App = () => (
+class App extends React.Component {
```
Comment (slaweet) [inline src/main.js]: this file (`src/main.js`) is not included in the test coverage report. Apparently, the report covers only files inside at least second level folders. This file has no tests

Comment (reyraa) [inline src/main.js]: This tic ket is not related to this file

**Comment by Isabello** [comment]
Can fix this. I think is a bug in the way it is executing eslint

File: src/utils/polyfills.test.js (hunk 2 above)
Comment (reyraa) [inline src/utils/polyfills.test.js]: Done
